ocs_ci/helpers/dr_helpers.py

```diff
@@ -799,16 +803,34 @@ def wait_for_replication_resources_creation(
         logger.error(error_msg)
         raise TimeoutExpiredError(error_msg)
 
+    if mix_workload:
+        resource_map = {
+            "rbd": (constants.VOLUME_REPLICATION, get_vr_count),
+            "cephfs": (constants.REPLICATION_SOURCE, get_replicationsources_count),
+        }
+
+        for key, vr_count in mix_workload_data.items():
+            if key not in resource_map:
```

Comment:
@prsurve Do we really need this because `mix_workload_data` dict is in user control, as we create this dictionary during deployment of the workload.